compiler/infra/OMRMonitor.hpp

```diff
@@ -47,14 +47,14 @@ class Monitor
    static TR::Monitor *create(char *name);
    static void destroy(TR::Monitor *monitor);
    void enter();
-   int32_t try_enter() { TR_UNIMPLEMENTED(); }
+   int32_t try_enter() { TR_UNIMPLEMENTED(); return 0; }
    int32_t exit(); // returns 0 on success
    void destroy();
    void wait() { TR_UNIMPLEMENTED(); }
-   intptr_t wait_timed(int64_t millis, int32_t nanos) { TR_UNIMPLEMENTED(); }
+   intptr_t wait_timed(int64_t millis, int32_t nanos) { TR_UNIMPLEMENTED(); return NULL; }
```

Comment:
Returning `NULL` here makes it seem like this function is returning a pointer.  Although the API for this function does not have any documentation in OMR, its use in downstream projects seems to suggest that this function returns a status code.

Can you return `0` instead?

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/dcec147dabf39419d0f87cf6f678d1ed2e9b6224..30a19e4b8c3a65c189c174ccdb3970ca868203d9).